code/obj/item/device/pda2/base_os.dm

```diff
@@ -696,24 +758,29 @@
 						else if (groupAddress in src.master.muted_mailgroups)
 							return
 
-					var/sender = signal.data["sender_name"]
-					if(!sender)
-						sender = "!Unknown!"
+					var/signalTag = signal.data["tag"]
+					if(signalTag in src.muted_alerts)
+						return
```

Comment:
Does this PR move the system which marks messages as alerts to tags? Just making sure.

Reply:
What about possibility of multiple tags?

Reply:
Originally, I had mailgroups identified via "group", and alerts via "tag", as previously there could be only one of each. But now that they can have multiple groups assigned to the same signal, it made more sense just to lump everything into the mailgroup check. And I forgot to change it until now. Should support multiple tags now.